qlty-check/src/cache.rs

```diff
@@ -271,7 +275,16 @@ impl IssuesCacheKey {
             cache_busters.insert(path, contents);
         }
 
+        let mut repository_sha: Option<String> = None;
+        let mut dirty_paths = Vec::new();
+        if let Ok(repository) = Workspace::new().and_then(|w| w.repo()) {
+            repository_sha = Self::repository_sha(&repository).ok();
+            dirty_paths = Self::collect_dirty_paths(&repository);
+        }
+
         Self {
+            repository_sha,
```

Comment:
Do we need to add the repository SHA to the digest also?

Reply:
This is what gets added to the digest.

Reply:
Ah thanks. I missed how add_target_sha was taking care of that.